src/nav/apm.yml

```diff
@@ -1,70 +1,13 @@
-              - title: Custom instrumentation
-                path: /docs/apm/agents/manage-apm-agents/agent-data/custom-instrumentation
-              - title: Agent attributes
-                path: /docs/apm/agents/manage-apm-agents/agent-data/agent-attributes
-              - title: Custom events
-                path: /docs/data-apis/custom-data/custom-events/apm-report-custom-events-attributes
-              - title: Custom metrics
-                path: /docs/apm/agents/manage-apm-agents/agent-data/collect-custom-metrics
-              - title: Manage errors
-                path: /docs/apm/agents/manage-apm-agents/agent-data/manage-errors-apm-collect-ignore-or-mark-expected
-              - title: Link Kubernetes
-                path: /docs/apm/agents/manage-apm-agents/agent-data/link-your-applications-kubernetes
-          - title: App naming
-            path: /docs/apm/agents/manage-apm-agents/app-naming
-            pages:
-              - title: Name your application
-                path: /docs/apm/agents/manage-apm-agents/app-naming/name-your-application
-              - title: Use multiple names for an app
-                path: /docs/apm/agents/manage-apm-agents/app-naming/use-multiple-names-app
-          - title: Configuration
-            path: /docs/apm/agents/manage-apm-agents/configuration
-            pages:
-              - title: Server-side configuration
-                path: /docs/apm/agents/manage-apm-agents/configuration/server-side-agent-configuration
-              - title: View config values for your app
-                path: /docs/apm/agents/manage-apm-agents/configuration/view-config-values-your-app
-              - title: High security mode
-                path: /docs/apm/agents/manage-apm-agents/configuration/high-security-mode
-              - title: Configurable security policies
-                path: /docs/apm/agents/manage-apm-agents/configuration/enable-configurable-security-policies
-              - title: 'Add, rename, and remove hosts'
-                path: /docs/apm/agents/manage-apm-agents/configuration/add-rename-remove-hosts
-              - title: Monitor background processes
-                path: /docs/apm/transactions/intro-transactions/monitor-background-processes-other-non-web-transactions
-              - title: Agent support for ARM64/Graviton and x86_64
-                path: /docs/apm/agents/manage-apm-agents/configuration/support-arm-graviton-x86-64
-          - title: Troubleshooting
-            path: /docs/apm/agents/manage-apm-agents/troubleshooting
-            pages:
-              - title: Configuration mismatch
-                path: /docs/apm/agents/manage-apm-agents/troubleshooting/agent-nrintegrationerrors-appear-insights
-              - title: Environment and services
-                path: /docs/apm/agents/manage-apm-agents/troubleshooting/get-environment-data-about-your-apm-app
-      - title: C SDK
+      - title: C SDK monitoring
```

Comment:
Is this the correct phrasing? Can you monitor an SDK?

Reply:
C SDK is EOL so I wouldn't worry about the wording here